subt/artf_node.py

```diff
@@ -145,6 +145,7 @@ def stdout(self, *args, **kwargs):
 
     def run(self):
         try:
+            self.stdout(cv2.dnn.getAvailableTargets(cv2.dnn.DNN_BACKEND_CUDA))
```

Comment:
We can remove this now.

Reply:
I would maybe leave it for local installations??

Reply:
ok